package/expo-package/src/optionalDependencies/takePhoto.ts

```diff
@@ -61,7 +61,7 @@ export const takePhoto = ImagePicker
               ...photo,
               cancelled: false,
               duration: photo.duration, // in milliseconds
-              name: 'video_recording_' + date + photo.uri.split('.').pop(),
+              name: 'video_recording_' + date + '.' + photo.uri.split('.').pop(),
```

Comment:
I really don't like how we sort of blindly do this - this can break on so many levels. Let's merge since it appears to never have worked but we need to fix this soon.